packages/lit-html/src/directive.ts

```diff
@@ -12,9 +12,17 @@ import {
   ElementPart,
 } from './lit-html';
 
-export type DirectiveClass = {
+export {
+  AttributePart,
+  BooleanAttributePart,
+  ChildPart,
+  EventPart,
+  Part,
+} from './lit-html';
```

Comment:
Hm, was there a reason why we didn't export the parts from directive.ts before?

I guess if they were exported from lit-html then they're not private, so I don't see any reason not to.

Reply:
before in lit1 it was because of a circular dependency which internal build can't handle. Idk why this doesn't trigger the same, but it don't

Reply:
I was just defaulting to exporting from one place.